pkg/controller/admissionchecks/multikueue/admissioncheck.go

```diff
@@ -115,7 +116,8 @@ func (a *ACReconciler) Reconcile(ctx context.Context, req reconcile.Request) (re
 	}
 
 	oldCondition := apimeta.FindStatusCondition(ac.Status.Conditions, kueue.AdmissionCheckActive)
-	if !cmpConditionState(oldCondition, &newCondition) {
+	if !ptr.Deref(ac.Status.SingleInstannceInClusterQueue, false) || !cmpConditionState(oldCondition, &newCondition) {
+		ac.Status.SingleInstannceInClusterQueue = ptr.To(true)
```

Comment:
Can we enforce `singleInstanceInClusterQueue=true` in advance like webhooks if the admissionChecks is for multikueue?

Reply:
No, the decision should be specific to multikueue, the initial implementation was based on a config list of controller names that need this but: https://github.com/kubernetes-sigs/kueue/pull/1635#discussion_r1466659290